### PR TITLE
Filling in README template

### PR DESCRIPTION
**Pull request by autodidaddict**
This pull request fills in the blanks from the original raw template with specifics to the blob store specification.

It also removes an unnecessary parameter from one of the wit files, as previously discussed with @Mossaka

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
-[Fill in the current phase, e.g. Phase 1]
+Phase 1
 
 ### Champions
 
-- [Champion 1]
-- [Champion 2]
-- [etc.]
+- Jiaxiao Zhou
+- Kevin Hoffman
```
Comment (Mossaka) [inline README.md:12]: I will be happy to have you as one of the champions of the wasi-blob-store proposal, @autodidaddict !

I will need to ask Lin / Pat about how to add one to the existing champions list.

Comment (pchickey) [reply README.md:12]: I asked Lin what the process here had been: we don't require any sort of WASI group process to change proposal champions beyond the existing champions agreeing to it. cc: @ricochet

**Comment by autodidaddict** [comment]
I think we're good to go on this for now. just awaiting an approval

**Review by ricochet — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Mossaka) [inline README.md:12]: Could you please add David Justice and Dan Chiarlone as the other two champions here?

**Review by Mossaka — APPROVED** [review]
Add one comment. Mostly LGTM!

File: README.md (hunk 1 above)
Comment (autodidaddict) [reply README.md:12]: Will do

**Comment by autodidaddict** [comment]
> Add one comment. Mostly LGTM!

Added the other champions.

**Pull request merged**